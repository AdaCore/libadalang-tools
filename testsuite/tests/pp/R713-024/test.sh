gnatpp -q --pipe spark_mode.ads
