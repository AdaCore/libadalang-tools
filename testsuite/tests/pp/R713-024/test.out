
WARNING: GNATpp will be baselined soon. Please switch to GNATformat, the new Ada code formatter, for future use.
-------

package Spark_Mode with
  SPARK_Mode
is
   pragma SPARK_Mode;
   procedure pre with
     Pre => True;
   use Spark_Mode;
   procedure spark_MODE;
end Spark_Mode;
